tsc/meetings/2019-01-17.md

```diff
@@ -0,0 +1,103 @@
+Minutes from 5th OpenVDB TSC meeting, Jan. 17, 2019
+
+Attendees: John M., *Jeff* L., *Nick* A., *Ken* M., *Peter* C., *Dan* B., Andrew P.
+
+Additional Attendees: Bruce Chernia (Intel), Daniel Heckenberg (Animal Logic)
+
+Agenda:
+
+1) Confirmation of quorum
+2) Selection of secretary
+3) Update from TAC (Ken)
+4) Update on Multi-Res grid from Autodesk (Ken)
+5) Intel representative
+6) JIT for OpenVDB (Nic)
+7) Copyright notice (Dan)
+8) Write permissions (Dan)
+9) Release process (Dan)
+10) Cleanup and improve repository (Ken)
+11) Schedule next meeting
+
+1) A quorum was confirmed.
+
+2) Secretary - Nick Avramoussis.
+
+3) OpenColourIO is accepted within the probationary stage. There's been interest form 3rd party
+companies, including Nvidia, Intel and Autodesk.
+
+Security and CI:
+
+The issue of a security expert was discussed as something which should be addressed across all
+ASWF projects. Jeff mentioned that the guidelines are vague for this role and that in the context
+of VDB, this mainly relates to bug handling/invalid file reading. Ken mentioned the concern of
+arbitrary metadata in regards to attacks. Andrew advised on static code analysis tools as part
+of a CI build (lgtm, clang). Dan mentioned using clang tidy as a CI step which would be easier
+to solve on the CMake side to address CI best practices.
```

Comment:
```suggestion
to solve on the CMake side to address CII best practices.
```

CII stands for "Core Infrastructure Initiative" (https://bestpractices.coreinfrastructure.org/en)